### PR TITLE
Law: acceleration due to gravity

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    cross_cartesian_vectors,
+    vector_magnitude,
+)
+
+# Description
+## The acceleration due to gravity is the acceleration a body in the gravity field of another body (e.g. Earth)
+## with rotational effects such as the centripetal acceleration taken into account. It is the same for all bodies
+## at a fixed point, but can have different magnitude and direction at different points in space.
+
+# Law: g = F_gravity / m - cross(w, cross(w, r))
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:23]: Maybe it's better to use it with centripetal acceleration symbol? And let user find proper way to calculate centripetal acceleration using any of the existing formulas.

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+## g - vector of acceleration due to gravity
+## F_gravity - vector of the force of gravity pull
+## w - pseudovector of angular velocity of rotation of moving frame S'
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:26]: What is moving frame? I do not see it in description.

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+## r - position vector in moving frame S'
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:27]: Is it immobile in the moving frame?
Position vector of what?

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:23]: I agree

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py [hunk 4]
```diff
@@ -16,15 +16,16 @@
...
-## with rotational effects such as the centripetal acceleration taken into account. It is the same for all bodies
-## at a fixed point, but can have different magnitude and direction at different points in space.
+## Suppose a reference frame S' is fixed to a rotating body A (e.g. Earth), so that frame S' rotates w.r.t.
+## another static reference frame S. The acceleration due to gravity (in moving frame S') is the
+## acceleration another body B has in the gravity field of body A, with rotational effects such as
+## the centripetal acceleration taken into account. It is the same for all bodies at a fixed point,
+## but can be different at different points in space.
 
-# Law: g = F_gravity / m - cross(w, cross(w, r))
+# Law: g = F_gravity / m - a_centripetal
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:25]: Now description does not correspond to implementation.

File: symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py [hunk 5]
```diff
@@ -16,15 +16,16 @@
...
 ## g - vector of acceleration due to gravity
 ## F_gravity - vector of the force of gravity pull
-## w - pseudovector of angular velocity of rotation of moving frame S'
-## r - position vector in moving frame S'
+## a_centripetal - vector of centripetal acceleration
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/vector/acceleration_due_to_gravity_via_gravity_force_and_centripetal_acceleration.py:28]: Acceleration of what?

**Review by blackyblack — APPROVED** [review]

**Pull request merged**